src/skspatial/objects/line.py

```diff
@@ -475,7 +475,7 @@ def distance_line(self, other: Line) -> np.float64:
 
         return distance
 
-    def intersect_line(self, other: Line, **kwargs) -> Point:
+    def intersect_line(self, other: Line, tol: np.float64 = None, **kwargs) -> Point:
```

Comment:
I don't see the point of accepting `tol` instead of just disabling the coplanar check. So I suggest adding a `check_coplanar` kwarg with default value `True`.
 
```suggestion
    def intersect_line(self, other: Line, check_coplanar: bool = True, **kwargs) -> Point:
```